Physics2D.Diagnostics/Diagnostics/DebugView.cs

```diff
@@ -143,6 +155,11 @@ private void DrawDebugData()
                     Transform xf = b.GetTransform();
                     foreach (Fixture f in b.FixtureList)
                     {
+                        if (EnableDebugFiltering)
+                            if (!_debuggableCategories.HasFlag(f.CollisionCategories))
```

Comment:
Generally I avoid HasFlag as it does additional checking at boxing the values.
Additionally I thing it returns true if the both values are zero which is not desirable in our case.
A simple bitwise operation will do the trick here. (_debuggableCategories & f.CollisionCategories) != 0.